src/main/java/com/openshift/restclient/capability/resources/IBuildTriggerable.java

```diff
@@ -30,7 +32,34 @@
 	 * Trigger a build with the given source level commit id
 	 * @param commitId
 	 * @return The build that was triggered
+	 * @deprecated
 	 */
 	IBuild trigger(String commitId);
+	
+	/**
+	 * Set the commit level for the git clone extraction
```

Comment:
ok, sure ... I replaced `git` with `SCM` and removed `clone` where used (perhaps also too git centric)